src/api/java/com/enderio/api/integration/Integration.java

```diff
@@ -0,0 +1,23 @@
+package com.enderio.api.integration;
+
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.forge.event.lifecycle.GatherDataEvent;
+
+public abstract class Integration implements IntegrationMethods{
```

Comment:
Not to be a pedant, but there should be a space between IntegrationMethods and the {